www/layouts/newsletter/section.html

```diff
@@ -447,5 +447,5 @@ <h2>Be informed</h2>
   </script>
   <!--End mc_embed_signup-->
 </div>
-
+{{ block "footer" . }} {{ partialCached "footer" . }} {{end}}
```

Comment:
I don't recall ever using a "newsletter" layout. Is there a screenshot for this?

Reply:
Let me share those with you, this was one of my first PR's on this project

<img width="1440" alt="Screenshot 2022-03-31 at 5 18 33 PM" src="https://user-images.githubusercontent.com/87968618/161053240-fe8b8be8-91b5-4ac4-b386-a9252c4624d2.png">
<img width="1440" alt="Screenshot 2022-03-31 at 5 18 25 PM" src="https://user-images.githubusercontent.com/87968618/161053277-34805dd7-4ada-4e5a-b6a2-b23ef1f6363b.png">

Reply:
of course, I remember now! 🙂